### PR TITLE
Sonoflow Sampling at 20Hz and averaging every second

### PR DESCRIPTION
**Pull request by Vitara-seddiki**

### CODE REVIEW

File: prod_dev/blood_gas_board/src/blood_entry.hpp [hunk 1]
```diff
@@ -101,7 +101,7 @@ static constexpr uint8_t MIN_THRESHOLD = 0;
 static constexpr uint16_t START_DELAY  = 200 * 30;
 
 /// @brief Sonoflow device read time period
-static constexpr int sonoflow_read_period_ms = 100;
+static constexpr int sonoflow_read_period_ms = 50;  //Ximedica 094 value was 100
```
Comment (vitara-srinivasan) [inline prod_dev/blood_gas_board/src/blood_entry.hpp:104]: The value 100 was introduced in VBI 0.10.1

File: prod_dev/blood_gas_board/src/blood_entry.hpp [hunk 2]
```diff
@@ -86,10 +86,10 @@ static constexpr uint8_t NUM_SAMPLES_TO_AVERAGE = 2;
 static constexpr uint8_t NUM_SAMPLES_TO_CALIBRATE = 4;
 
 /// @brief maintain the time interval between messages sending to UI
-static constexpr auto TIME_BETWEEN_SENDING = TX_TIMER_TICKS_PER_SECOND;
+static constexpr auto TIME_BETWEEN_SENDING = TX_TIMER_TICKS_PER_SECOND;  //Ximedica 094 value Every second
 
 /// @brief maintain the time interval between messages sending to UI
-static constexpr auto TIME_BETWEEN_LOGGING_MSG = (TX_TIMER_TICKS_PER_SECOND/2);
+static constexpr auto TIME_BETWEEN_LOGGING_MSG = (TX_TIMER_TICKS_PER_SECOND/2); //Ximedica 094 valueevery 500 ms
```
Comment (vitara-srinivasan) [inline prod_dev/blood_gas_board/src/blood_entry.hpp:92]: new value. not in 0_9_4

File: prod_dev/blood_gas_board/src/blood_entry.hpp (hunk 1 above)
Comment (Vitara-seddiki) [reply prod_dev/blood_gas_board/src/blood_entry.hpp:104]: Fixed as indicated.

File: prod_dev/blood_gas_board/src/blood_entry.hpp (hunk 2 above)
Comment (Vitara-seddiki) [reply prod_dev/blood_gas_board/src/blood_entry.hpp:92]: Fixed as indicated

**Review by vitara-campbell — COMMENTED** [review]
look fine

**Pull request merged**